docs/api/instruments/rohdeschwarz/sfm.rst

```diff
@@ -0,0 +1,11 @@
+######################################
+R&S SFM TV test transmitter
+######################################
+
+.. autoclass:: pymeasure.instruments.rohdeschwarz.SFM
+    :members:
```

Comment:
I think `pymeasure.instruments.rohdeschwarz.SFM` should be replaced by `pymeasure.instruments.rohdeschwarz.sfm.SFM`, but I have checked and Sphinx works with both.
I would do the change anyway for coherence with other `autoclass` directives.

Reply:
THey way you describe it, seems to be the more common way.
I will change it for coherence after I repaired the damage from my rebase issue

Reply:
changed it to pymeasure.instruments.rohdeschwarz.sfm.SFM